tests/RobotFramework/tests/tedge/diag/predefined_plugins.robot

```diff
@@ -0,0 +1,71 @@
+*** Settings ***
+Library             String
+Library             ThinEdgeIO
+
+Suite Setup         Custom Suite Setup
+Suite Teardown      Get Suite Logs
+
+Test Tags           theme:troubleshooting    theme:cli    theme:plugins
+
+
+*** Test Cases ***
+01_tedge
+    ${log_names}=    Create List
+    ...    output.log
+    ...    tedge-agent.log
+    ...    tedge-mapper-c8y.log
+    ...    tedge-mapper-collectd.log
+    ...    tedge-config-list.log
+    ...    tedge.toml
+    FOR    ${log_name}    IN    @{log_names}
+        File Size Is Not Zero    ${log_name}
```

Comment:
Since these are the dedicated tests for each plugin, I'd go one step further and assert at least some content: like the first line or some known/guaranteed content, just to be sure that what's captured is not some junk.

Reply:
All your suggestions are applied in e2af4a12ef684b014577d417eb64c3281be914c6